internal/nodes/avi_vrf_translator.go

```diff
@@ -97,10 +97,13 @@ func (o *AviObjectGraph) BuildVRFGraph(key, vrfName, nodeName string, deleteFlag
 			diff := lenNewNodeRoutes - nodeStaticRouteDetails.Count
 
 			var staticRouteCopy []*models.StaticRoute
-			staticRouteCopy = append(staticRouteCopy, aviVrfNode.StaticRoutes[:startIndex]...)
+			if startIndex <= len(aviVrfNode.StaticRoutes) {
```

Comment:
Can `startIndex` be negative?

Reply:
Should we have some logs to indicate record is not added?

Reply:
We can add a message showing index is either -ve or out of length but not sure if we would be able to say specifically this record is not added, since if we would try to obtaibe value at that index, it will again get into panic state

Reply:
Done